samples/Functions-ReplyToDiscussions/HttpTrigger1.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace MyAmbassadorDemo.Function
+{
+    public static class HttpTrigger1
```

Comment:
Do we need this file when we already have timmer

Reply:
Removed HttpTrigger